pkg/get-canary.go

```diff
@@ -0,0 +1,39 @@
+package pkg
+
+import (
+	"os"
+    "os/exec"
+
+	"github.com/flanksource/commons/logger"
+	"github.com/spf13/cobra"
+
+)
+
+var GetCanary = &cobra.Command{
+		Use: "get-canary",
+		Short: "Get canary information",
+		Args:  cobra.MinimumNArgs(0),
+		Run: func(cmd *cobra.Command, args []string) {
+			logger.Infof(`Retrieving Canary Info...
+				`)
+			Retrieva()			 
+		},
+	}
+
+func Retrieva() {
```

Comment:
This method should be called by api.Serve method and replace the inline canary data

Reply:
Noted